### ISSUE TITLE
Fails to load on safari

### ISSUE THREAD

**Issue by o1lab** [nocodb/nocodb#799]

**Comment by mccurdyc** [comment]
+1

First, thank you for the awesome tool you've built!

I've experience issues loading on iPadOS (15.2) in both Safari and Firefox 39.0 with NocoDB version 0.82.0.

```
docker run -d --name nocodb -p 8080:8080 -v $(pwd)/db-data:/usr/app/data/ nocodb/nocodb:0.82.0
```

In `docker logs` I see the following just infinitely repeating.

```
POST /dashboard/?q=sqlOp_xcPluginRead 200 - - 5.189 ms
GET /dashboard/auth/type 304 - - 6.060 ms
POST /dashboard/?q=sqlOp_xcRelease 200 74 - 0.734 ms
GET /dashboard/_nuxt/30f6081.js 304 - - 10.868 ms
GET /dashboard/_nuxt/12cc992.js 304 - - 11.748 ms
GET /dashboard/_nuxt/e56996b.js 304 - - 30.620 ms
GET /dashboard/_nuxt/e1bfbcc.js 304 - - 36.939 ms
GET /dashboard/_nuxt/48c29c4.js 200 983415 - 47.615 ms
GET /dashboard/ 200 2596 - 1.437 ms
```

I am able to get through to the signup page some times, but when I do and I create an account, I can't then sign in (maybe a separate issue).

**Comment by pranavxc** [comment]
Fixed in `v0.83.0`
